# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Clear11.cpp
# Commit: 040a6d3c401d
# Full Hash: 040a6d3c401de4142568a0c942f169244dc72a71
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-12 09:34:21
# Description:
#   Bug 1636122 - [angle] Use deviceContext instead of deviceContext1. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74770
# ==============================================================================

diff -r 9923799dd260 -r 040a6d3c401d gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Clear11.cpp
--- a/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Clear11.cpp	Tue May 12 01:13:17 2020 +0000
+++ b/gfx/angle/checkout/src/libANGLE/renderer/d3d/d3d11/Clear11.cpp	Tue May 12 01:22:38 2020 +0000
@@ -521,7 +521,7 @@
             UINT sampleMask;
             ID3D11BlendState *blendState;
             D3D11_BLEND_DESC blendDesc;
-            deviceContext1->OMGetBlendState(&blendState, blendFactor, &sampleMask);
+            deviceContext->OMGetBlendState(&blendState, blendFactor, &sampleMask);
             if (blendState) {
                     blendState->GetDesc(&blendDesc);
                     // You can only use dual source blending on slot 0 so only check there